.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: "\U0001F41B Bug Report"
 about: Report a bug
-title: "[module] "
+title: "(insert module name): short issue description"
```

Comment:
Better than just `module`, but seems a little wordy. Perhaps `module name` would be better